code/controllers/subsystem/mapping.dm

```diff
@@ -294,7 +294,7 @@ Used by the AI doomsday and the self-destruct nuke.
 	z_list = SSmapping.z_list
 	multiz_levels = SSmapping.multiz_levels
 
-#define INIT_ANNOUNCE(X) to_chat(world, span_debug("[X]")); log_world(X)
+#define INIT_ANNOUNCE(X) to_chat(world, X); log_world(X)
```

Comment:
Don't make these debug spans, I like the way it looks as-is.

Reply:
This used the old debug span as well.